packages/manager/modules/vps/src/dashboard/vps-dashboard.routing.js

```diff
@@ -118,6 +118,9 @@ export default /* @ngInject */ ($stateProvider) => {
           upgradeType,
         }),
 
+      adviceGroup: /* @ngInject */ (serviceName, VpsService) =>
```

Comment:
don't load advices in resolve, it blocks page load. We don't want to block page load while loading advice. Advice must be loaded lazily. Use a separate advice component and load devices in a controller. Refer to https://github.com/ovh/manager/pull/4218/files